<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Genesys Reader EEP Generated Tool</string>
  </property>
  <widget class="QPushButton" name="generatedbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generated EEP File</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>USB VID/PID Setting:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Product ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vid">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>05E3</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="pid">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0748</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>351</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string>Lun/Icon Setting:</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lun Number Disp.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Physical Card Support:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcs_ms">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MS (Memory Stick)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcs_sd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SD / MMC / EMMC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcs_cf">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CF (Compact Flash)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcs_msd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>microSD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcs_m2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>M2 (Memory Stick Micro)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcs_xd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>xD (Extreme Digital)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lun Assignment:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LUN0 (Icon 0):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="mmc_support">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MMC support</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lun0_assignment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MS (Memory Stick)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD / MMC / EMMC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CF (Compact Flash)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>microSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2 (Memory Stick Micro)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xD (Extreme Digital)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="lun1_assignment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>MS (Memory Stick)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD / MMC / EMMC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CF (Compact Flash)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>microSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2 (Memory Stick Micro)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xD (Extreme Digital)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LUN1 (Icon 1):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="lun2_assignment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>MS (Memory Stick)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD / MMC / EMMC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CF (Compact Flash)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>microSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2 (Memory Stick Micro)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xD (Extreme Digital)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LUN2 (Icon 2):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="lun3_assignment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>MS (Memory Stick)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD / MMC / EMMC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CF (Compact Flash)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>microSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2 (Memory Stick Micro)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xD (Extreme Digital)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LUN3 (Icon 3):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="lun4_assignment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>MS (Memory Stick)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD / MMC / EMMC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CF (Compact Flash)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>microSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2 (Memory Stick Micro)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xD (Extreme Digital)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LUN4 (Icon 4):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="lun5_assignment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>MS (Memory Stick)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD / MMC / EMMC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CF (Compact Flash)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>microSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2 (Memory Stick Micro)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xD (Extreme Digital)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LUN5 (Icon 5):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>331</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(146, 146, 146);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Hardware Setting:</string>
   </property>
   <widget class="QCheckBox" name="u1u2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>U1/U2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lpm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LPM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="power_saving_mode">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Power-Saving Mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="remote_wakeup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote WakeUp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ssc">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SSC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ltm">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LTM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="readonly">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read-Only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="customized_string">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Customized String</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>USB Power Setting:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="usb_power_setting">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>900</number>
    </property>
    <property name="singleStep">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Customized String Setting:</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendor String:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="v_str">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Genesys</string>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="p_str">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generic USB</string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Product String:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s_str">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0000000000</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial String:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Card String:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_str_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SD</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_str_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_str_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_str_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>LED Setting:</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="led_mode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Access LED is Card Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Access LED is Power Access</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>LED GPIO Setting For GL3223 Only:</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pin Defined:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="_3233_pin_defined">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>#48: SS Act; #128: HS Act</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>#48: LED Mode; #128: Power</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="sd2_wp_enabled">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable #128 as SD2_WP</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>660</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Powered by barryblueice, 2025. &lt;br/&gt;NOT Genesys Official.&lt;br/&gt;OpenSource-Link: &lt;a href=&quot;https://github.com/barryblueice/Genesys-Reader-EEP-Generated-Tool&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#003e92;&quot;&gt;GitHub-Genesys Reader EEP Generated Tool&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
